pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/api/JavaOperationMetricKey.java

```diff
@@ -20,6 +21,13 @@
  */
 public enum JavaOperationMetricKey implements MetricKey<MethodLikeNode> {
 
+    /**
+     * Cognitive complexity.
+     *
+     * @see CognitiveComplexityMetric
+     */
+    COGNITIVE(new CognitiveComplexityMetric()),
```

Comment:
Please call this constant `COGNITIVE_COMPLEXITY`. I think using shortened names for these constants was a mistake, and new ones should be explicit.